.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 .phpunit.result.cache
+.idea
```

Comment:
Please add the files you wish to ignore from your editor of choice to your global .gitignore file

Reply:
I don't want the .idea folder to be always ignored. In some project I do want code style files in the .idea folder to be shared amongst colleagues. I thought this would made it easier for other developers to contribute to the package. I have removed the line. Thanks